### ISSUE TITLE
To enable pressure to be plotted as altitude variable (for develop_aircraft plots), requires some additional coding.

### ISSUE THREAD

**Issue by quaz115** [NOAA-CSL/MELODIES-MONET#187]
This suggests that pressure can also be plotted but the labeling in make_vertprofile() is currently all specific to altitude in meters. Would be nice to add in plotting by pressure in the future, but it would require some additional coding such as flipping the y axis to start from high values and decrease as it goes up. For now I would remove the commented line and say that altitude must be in meters after unit_scale application on the variable.

_Originally posted by @colin-harkins in https://github.com/NOAA-CSL/MELODIES-MONET/pull/181#discussion_r1294923974_

**Comment by L12D2** [comment]
Please refer to the most recent develop push to see these changes 

    if altitude_variable == "pressure_obs":
        y0, y1 = ax.get_ylim()
        if y0 < y1:
            ax.invert_yaxis()

**Comment by L12D2** [comment]
@rschwant Was going through some of the MM issues, and this is a feature that I think should be marked as "done" / in v1.1. I have actually been using this feature for the ASIA-AQ data

**Comment by rschwant** [comment]
Yes @L12D2 you are correct this now works for the secondary y-axis of the timeseries plot and it also works for the vertical profile plot.

We should keep this open because I'm not sure that you can plot with altitude for the curtain plot. I think the curtain plot right now is hardcoded to only use pressure.
